eventsub-common/src/main/java/com/github/twitch4j/eventsub/events/ChannelPollEndEvent.java

```diff
@@ -20,4 +22,14 @@
      */
     private PollStatus status;
 
+    /**
+     * The time the poll ended.
+     */
+    private Instant endedAt;
+
+    @Override
+    public Instant getEndsAt() {
```

Comment:
Should this field be removed from the shared event class (`ChannelPollEvent`)—or is this fine?

Reply:
I think it's cleaner to remove it from `ChannelPollEvent`.